src/files-card-body.jsx

```diff
@@ -259,6 +259,14 @@ export const FilesCardBody = ({
                 });
             } else if (e.key === "Enter" && selected.length === 1) {
                 onDoubleClickNavigate(selected[0]);
+            } else if (e.key === "Delete" && selected.length !== 0) {
```

Comment:
This added line is not executed by any test.

Reply:
HMMM! It should be!